app/settings.py

```diff
@@ -154,3 +154,5 @@ def utcoffset_or_fail(date_value, key):
 )
 
 ONS_URL = os.getenv("ONS_URL", "https://www.ons.gov.uk")
+
+RAS_URL = os.getenv("ONS_URL", "https://surveys.ons.gov.uk")
```

Comment:
Is this supposed to be RAS_URL?
```suggestion
RAS_URL = os.getenv("RAS_URL", "https://surveys.ons.gov.uk")
```